package.json

```diff
@@ -1,7 +1,8 @@
 {
   "nyc": {
     "exclude": [
-      "apis"
+      "apis",
+      "test"
```

Comment:
Sorry for the late comment, but I think this should be `"build/test"` to property excluded, isn't it?